lib/semantics/mod-file.cc

```diff
@@ -437,13 +437,13 @@ void PutTypeParam(std::ostream &os, const Symbol &symbol) {
 
 void PutInit(std::ostream &os, const MaybeExpr &init) {
   if (init) {
-    init->AsFortran(os << '=');
+    os << '=' << init;
```

Comment:
For writing .mod files I prefer staying with the `AsFortran` call. It provides a more precise guarantee about what it does: write the expression as Fortran. Whereas `operator<<` just means output this expression somehow.

Reply:
OK.